src/sagas/driveTimerSaga.js

```diff
@@ -0,0 +1,19 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+
+import { getDriveTimer } from 'apis/driveTimerApi';
+import { driveTimerAction, driveTimerRequestFailed } from 'actions/driveTimerActions';
+
+import { DRIVE_TIMER } from 'constants/actionConstants';
+
+export function* driveTimer() {
+  try {
+    const { data } = yield call(getDriveTimer);
+    yield put(driveTimerAction(data));
+  } catch (error) {
+    yield put(driveTimerRequestFailed('Something went wrong with drive timer!'));
+  }
+}
+
+export default function* driveTimerSaga() {
```

Comment:
Codacy has a fix for the issue: [Missing space before *.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
export default function * driveTimerSaga() {
```